ecs-cli/modules/compose/cli/ecs/app/service_command.go

```diff
@@ -136,3 +136,36 @@ func deploymentConfigFlags(specifyDefaults bool) []cli.Flag {
 		},
 	}
 }
+
+func loadBalancerFlags(specifyDefaults bool) []cli.Flag {
```

Comment:
Just FYI, the text is copy pasted from the param description in the AWS API docs.